src/main/java/com/mcmoddev/lib/util/Tuple2.java

```diff
@@ -0,0 +1,11 @@
+package com.mcmoddev.lib.util;

```

Comment:
Wouldn't the Apache Commons implementation be better? If memory serves, it is part of the default libraries shipped with MC/Forge

Reply:
See: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/package-summary.html